### PR TITLE
Change wording regarding app.RunOaktonCommands vs. App.Run

### PR DESCRIPTION
**Pull request by rigtigeEmil**

### CODE REVIEW

File: docs/configuration/cli.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ builder.Host.ApplyOaktonExtensions();
 <sup><a href='https://github.com/JasperFx/marten/blob/master/src/samples/MinimalAPI/Program.cs#L9-L17' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_using_webapplication_1' title='Start of snippet'>anchor</a></sup>
 <!-- endSnippet -->
 
-And finally, use Oakton as the command line parser and executor with this last line of code in your
+And finally, use Oakton as the command line parser and executor by replacing `App.Run()` with this line of code in your
```
Comment (oskardudycz) [inline docs/configuration/cli.md:58]:
```suggestion
And finally, use Oakton as the command line parser and executor by replacing `App.Run()` as the last line of code in your
```

**Comment by oskardudycz** [comment]
Thank you!

**Comment by mysticmind** [comment]
@oskardudycz one of the changes is within the code sample so updating cli.md would get overridden when our mdsnippets runs. So it is better to change in the code.

**Comment by oskardudycz** [comment]
@mysticmind, right, my bad, didn’t notice that.

**Comment by oskardudycz** [comment]
@mysticmind done it in https://github.com/JasperFx/marten/commit/bacd4267bc3e41105923c2244abb9fa99afd1708.

**Comment by mysticmind** [comment]
> @mysticmind done it in [bacd426](https://github.com/JasperFx/marten/commit/bacd4267bc3e41105923c2244abb9fa99afd1708).

Thanks @oskardudycz!

**Pull request merged**